src/components/textarea-with-counter.test.tsx

```
@@ -0,0 +1,53 @@
+import { describe, expect, it } from 'vitest'
+import { render, screen } from '@testing-library/react'
+import { TextareaWithCounter } from './textarea-with-counter'
```

ESLint

[error] `./textarea-with-counter` import should occur after import of `@testing-library/user-event`

(import/order)